test/e2e/network/pod_lifecycle.go

```diff
@@ -81,19 +83,18 @@ var _ = common.SIGDescribe("Connectivity Pod Lifecycle", func() {
 	})
 
 	ginkgo.It("should be able to connect to other Pod from a terminating Pod", func(ctx context.Context) {
-		ginkgo.By("Creating 1 webserver pod able to serve traffic during the grace period of 300 seconds")
-		gracePeriod := int64(100)
```

Comment:
leave this as it was per test, no need to consolidate them, also fix the comment that says 300 seconds, don;t change to `longNetworkGracePeriod`

Reply:
Yes, that was one of the main purposes of this PR, there is this inconsistency due to hardcoded values, thanks for understanding. I will do the changes.